presto-main/src/main/java/com/facebook/presto/execution/SqlQueryExecution.java

```diff
@@ -106,6 +107,7 @@
     private final ExecutionPolicy executionPolicy;
     private final List<Expression> parameters;
     private final SplitSchedulerStats schedulerStats;
+    private final AtomicReference<ResourceGroupId> resourceGroupIdAtomicReference = new AtomicReference<>();
```

Comment:
Could you please explain what's the naming convention? Why we should use ResourceGroup here instead of resouceGroupID or recourceGroupIdAtomicReference? Thanks!